src/V2/player.ts

```diff
@@ -79,6 +79,7 @@ export default class Player {
     private fileHeader: FileHeader | null
     private vertexShader: string
     private fragmentShader: string
+    //@ts-ignore
     private intervalId: NodeJS.Timer
```

Comment:
Isn't that just a number? Why overcomplicate it twice?

Reply:
Yes. I'll make it a number.